SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1789,23 +1807,23 @@ void ActorSystem::RemoveActor(const Worker_EntityId EntityId)
 		}
 	}
 
-	// Actor is a startup actor that is a part of the level. If it's not Tombstone'd, then it
+	// Actor is a startup actor that is a part of the level. If it's not Tombstone-d, then it
 	// has just fallen out of our view and we should only remove the entity.
 	if (Actor->IsFullNameStableForNetworking() && !ActorSubView->HasComponent(EntityId, SpatialConstants::TOMBSTONE_COMPONENT_ID))
 	{
-		NetDriver->PackageMap->ClearRemovedDynamicSubobjectObjectRefs(EntityId);
+		ClearRemovedDynamicSubObjectRefs(EntityId);
 		if (USpatialActorChannel* Channel = NetDriver->GetActorChannelByEntityId(EntityId))
 		{
-			for (UObject* DynamicSubobject : Channel->CreateSubObjects)
+			for (UObject* DynamicSubObject : Channel->CreateSubObjects)
```

Comment:
In the future, please avoid these kind of changes in medium-big PRs, it makes it hard to know what change has been made on these specific lines.

Reply:
yup, I'm looking at the diff and you're right, it is noisy! Will avoid in future